CIP-0036/README.md

```diff
@@ -55,12 +55,28 @@ Each delegation therefore contains:
   - a voting key: simply an ED25519 public key. This is the spending credential in the sidechain that will receive voting power from this delegation. For direct voting it's necessary to have the corresponding private key to cast votes in the sidechain. How this key is created is up to the wallet.
   - the weight that is associated with this key: this is a 4-byte unsigned integer (CBOR major type 0, The weight may range from 0 to 2^32-1) that represents the relative weight of this delegation over the total weight of all delegations in the same registration transaction.
 
-### Voting key derivation path
+### Voting key 
+
+The terms voting keys and vote keys should be used interchangeably to indicate the keys described in this specification. The term governance should not be associated with these keys.
```

Comment:
Is the vote/voting dichotomy worth it? For talking about in natural language it's OK, but for code it's not great if you have two ways of naming the same thing. So in the section "Tooling" I'd propose to only have one thing, or one of them strongly preferred (I'd go for the shorter version if other reasons are absent).

Otherwise, anyone working with two tools that use different names, will have to choose one of the ways, and then his code will be inconsistent with the other tool (and everyone dealing with that code will have to wonder "why are we using vote while the tool uses voting ". And people will needlessly get stupid bugs when integrating tools/libraries and mistakenly use one name instead of the other.

[A mess of this kind was created by reward account / reward address / stake address / staking address for that particular type of Cardano address. Every time someone new comes to the ecosystem, he has to think about whether these different names refer to the same thing or not, and ask around for clarification.]

Reply:
@janmazak 

Thanks for your suggestion - you raise a good point, I agree. I will change the wording to prefer the term "vote" for tooling.